prisma/schema.prisma

```diff
@@ -351,6 +351,7 @@ model RealmHistory {
   eventId         String
   eventType       String
   realmId         Int
+  bastionId       Int        @default(0)
```

Comment:
should bastions be a field of a realm history? or should they be its own history.

Reply:
the bastion history is linked to the realms history in the sense that all the events in the bastion should also be in the RealmHistory. And the combat and travel events from RealmHistory should also be shown in the BastionHistory. 

After giving it more thought i think that this would be a better way to do it:
```
model BastionHistory {
  bastionId      Int          
  realmHistory   RealmHistory @relation(fields: [realmHistoryId], references: [id])
  realmHistoryId Int               @id

  @@unique([realmHistoryId])
}

```

So every time there is a new event it will be added to RealmHistory. And for the events linked to a bastion, there is another db with only a bastionId and a reference to that event in RealmHistory.

Reply:
i've been using eventId as the unique id on all indexed things. as we're syncing from chain. eventId = blockNumber_transactionIndex_eventIndex, so the chain was source of truth, and could upsert fields when we made changes, and didn't have to worry about internal autoincrement keys from local db.

realm history primary key is:
`@@unique([eventId, eventType])`

Reply:
good point. 

Here's what i propose then:
```
model BastionHistory {
  bastionId      Int          
  realmHistory   RealmHistory @relation(fields: [realmHistoryEventId, realmHistoryEventType], references: [eventId, eventType])
  realmHistoryEventId String               
  realmHistoryEventType String             

  @@unique([realmHistoryEventId, realmHistoryEventType])
}

```

Reply:
👍